frontend/packages/shared/src/components/GiteaHeader/ThreeDotsMenu/LocalChangesModal/LocalChanges/LocalChanges.module.css

```diff
@@ -0,0 +1,12 @@
+.contentWrapper {
+  padding: 40px;
+  width: calc(var(--modal-max-width) - var(--modal-border-size));
+  height: calc(
+    var(--modal-content-max-height) - var(--heading-height) - var(--modal-border-radius)
+  );
```

Comment:
Er det nødvendig å kalkulere størrelsen på denne måten? Har du prøvd å løse det ved hjelp av `flex`? Her er en nyttig guide som jeg selv ofte er innom: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
der var forslag fra Andrea 😊

Reply:
fjernet alle `calc `og brukte vanlig verdier til `height `og `width`

Reply:
I siste endringer , brukte noe som ligner på `settingsModal `slik at det blir ikke stor forskjell mellom modaler i localchsages og setting. og da ble brukt bare en `calc`